compiler/z/codegen/OMRInstruction.hpp

```diff
@@ -144,8 +144,7 @@ class OMR_EXTENSIBLE Instruction : public OMR::Instruction
    void setDebugHookOp() {_flags.set(DebugHookOp);}
 
    bool hasLongDisplacementSupport();
-   TR::InstOpCode::Mnemonic opCodeCanBeAdjustedTo(TR::InstOpCode::Mnemonic);
-   void attemptOpAdjustmentForLongDisplacement();
+   static TR::InstOpCode::Mnemonic opCodeCanBeAdjustedTo(TR::InstOpCode::Mnemonic);
```

Comment:
In subsequent PRs I think I want to rename this and probably push it out to OMRInstOpcode.cpp.